### ISSUE TITLE
Spectra export

### ISSUE THREAD

**Issue by JB-MS** [chhh/batmass#10]
Hi there,

I was wondering if there is any possibility for export of spectra as e.g. svg or pdf? Is this feature planned?

Cheers,
Johannes

**Comment by rsalek** [comment]
Good suggestion. I usually take a snapshot. but having such functionality would be nice.

**Comment by chhh** [comment]
All the viewers need to be overhauled significantly, I just don't have the time to do that. But it's definitely the plan.

**Comment by JB-MS** [comment]
I am definitely familiar with the time issue...

A bit unrelated, but i don't want to spam you with more issues: Is it (maybe quickly) possible to implement reading of gzipped files?

Cheers,
Johannes

**Comment by chhh** [comment]
@JB-MS not really, for gzipped files a linear read through the whole file is required every time as you need to open a decompressing stream. The underlying reader library (mdftbx)[https://github.com/chhh/msftbx] doesn't support that as I originally didn't think everything through, it was made for random access. In the next iteration there will be 2 interfaces - for random access and for linear reads, then it will be possible.
